   LOG VENPRO 2018-02-18
_________________________________________________________
---------------------------------------------------------

2018-02-18 23:36:49 : VENPRO iniciado
2018-02-18 23:37:50 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20180218. System.Net.WebException: Error en el servidor remoto: (550) Archivo no disponible (ej. no se encuentra el archivo o no se tiene acceso).
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.SyncRequestCallback(Object obj)
   en System.IO.Stream.Close()
   en System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   en System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   en System.Net.FtpWebRequest.FinishRequestStage(RequestStage stage)
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:38:5 : Error GetFileList(). No se pudo listar archivos - ftp://nuevapc//107/pcmaster/20180218. System.Net.WebException: No se puede resolver el nombre remoto: 'nuevapc'
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:38:10 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20180218. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:38:51 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20180218. System.Net.WebException: Error en el servidor remoto: (550) Archivo no disponible (ej. no se encuentra el archivo o no se tiene acceso).
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.SyncRequestCallback(Object obj)
   en System.IO.Stream.Close()
   en System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   en System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   en System.Net.FtpWebRequest.FinishRequestStage(RequestStage stage)
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:39:5 : Error GetFileList(). No se pudo listar archivos - ftp://nuevapc//107/pcmaster/20180218. System.Net.WebException: No se puede resolver el nombre remoto: 'nuevapc'
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:39:9 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20180218. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:39:48 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20180218. System.Net.WebException: Error en el servidor remoto: (550) Archivo no disponible (ej. no se encuentra el archivo o no se tiene acceso).
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.SyncRequestCallback(Object obj)
   en System.IO.Stream.Close()
   en System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   en System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   en System.Net.FtpWebRequest.FinishRequestStage(RequestStage stage)
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:40:3 : Error GetFileList(). No se pudo listar archivos - ftp://nuevapc//107/pcmaster/20180218. System.Net.WebException: No se puede resolver el nombre remoto: 'nuevapc'
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:40:9 : Error GetFileList(). No se pudo listar archivos - ftp://192.168.52.141//104/pcmaster/20180218. System.Net.WebException: No es posible conectar con el servidor remoto
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:40:48 : Error GetFileList(). No se pudo listar archivos - ftp://localhost//103/pcmaster/20180218. System.Net.WebException: Error en el servidor remoto: (550) Archivo no disponible (ej. no se encuentra el archivo o no se tiene acceso).
   en System.Net.FtpWebRequest.CheckError()
   en System.Net.FtpWebRequest.SyncRequestCallback(Object obj)
   en System.IO.Stream.Close()
   en System.Net.ConnectionPool.Destroy(PooledStream pooledStream)
   en System.Net.ConnectionPool.PutConnection(PooledStream pooledStream, Object owningObject, Int32 creationTimeout, Boolean canReuse)
   en System.Net.FtpWebRequest.FinishRequestStage(RequestStage stage)
   en System.Net.FtpWebRequest.GetResponse()
   en VENPRO.globales.GetFileList(String rutaserver, String user, String pass) en E:\VENPRO\VENPRO\globales.cs:línea 564
2018-02-18 23:41:1 : VENPRO Cerrado.
